pipelines/multiprofit_fit_standard.yaml

```diff
@@ -5,13 +5,20 @@ parameters:
   fix_centroid: False
   use_shapelet_psf: False
 tasks:
-  fitDeepCoaddPsfGaussians: lsst.meas.extensions.multiprofit.pipetasks_fit.MultiProFitCoaddPsfFitTask
+  fitDeepCoaddPsfGaussians:
+    class: lsst.meas.extensions.multiprofit.pipetasks_fit.MultiProFitCoaddPsfFitTask
+    config:
+      use_cell_coadds: parameters.use_cell_coadds
   fitDeblendedObjectsSersic:
     class: lsst.meas.extensions.multiprofit.pipetasks_fit.MultiProFitCoaddSersicFitTask
     config:
+      use_cell_coadds: parameters.use_cell_coadds
```

Comment:
Is it recommended to rely on `parameters.use_cell_coadds` here without defining it, in the hopes that the pipeline importing this one defines it?

Reply:
I'm going to defer to @natelust on the implications of this.

Reply:
@natelust - any thoughts?

Also a question for @taranu - do you ever run this pipeline directly?

Reply:
Thanks, Nate! pexConfig catches it, and the before the error, there is a useful enough warning that says

```
UserWarning: config use_cell_coadds contains value parameters.use_cell_coadds which is formatted like a Pipeline parameter but was not found within the Pipeline, if this was not intentional, check for a typo
```

However, given that @taranu has not put this within a `_ingredients` folder, I am going to treat it as a standalone pipeline that should be runnable and define the parameter value, which the importing pipeline can overwrite.